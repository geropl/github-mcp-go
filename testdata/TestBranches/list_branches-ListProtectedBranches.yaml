---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 0
        transfer_encoding: []
        trailer: {}
        host: api.github.com
        remote_addr: ""
        request_uri: ""
        body: ""
        form: {}
        headers:
            Accept:
                - application/vnd.github.v3+json
            User-Agent:
                - go-github/v69.2.0
            X-Github-Api-Version:
                - "2022-11-28"
        url: https://api.github.com/repos/geropl/github-mcp-go-test/branches?per_page=100&protected=true
        method: GET
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: 2
        uncompressed: false
        body: '[]'
        headers:
            Access-Control-Allow-Origin:
                - '*'
            Access-Control-Expose-Headers:
                - ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
            Cache-Control:
                - private, max-age=60, s-maxage=60
            Content-Length:
                - "2"
            Content-Security-Policy:
                - default-src 'none'
            Content-Type:
                - application/json; charset=utf-8
            Date:
                - Fri, 07 Mar 2025 15:28:13 GMT
            Etag:
                - '"893ae4448e216dd56c14b24439abe5e2f6518c52a39a1c3b817e310f76469fdb"'
            Github-Authentication-Token-Expiration:
                - 2025-04-06 13:14:03 +0200
            Referrer-Policy:
                - origin-when-cross-origin, strict-origin-when-cross-origin
            Server:
                - github.com
            Strict-Transport-Security:
                - max-age=31536000; includeSubdomains; preload
            Vary:
                - Accept, Authorization, Cookie, X-GitHub-OTP,Accept-Encoding, Accept, X-Requested-With
            X-Accepted-Github-Permissions:
                - contents=read
            X-Content-Type-Options:
                - nosniff
            X-Frame-Options:
                - deny
            X-Github-Api-Version-Selected:
                - "2022-11-28"
            X-Github-Media-Type:
                - github.v3; format=json
            X-Github-Request-Id:
                - 2258:3C7982:43E225E:4603143:67CB108D
            X-Ratelimit-Limit:
                - "5000"
            X-Ratelimit-Remaining:
                - "4963"
            X-Ratelimit-Reset:
                - "1741364247"
            X-Ratelimit-Resource:
                - core
            X-Ratelimit-Used:
                - "37"
            X-Xss-Protection:
                - "0"
        status: 200 OK
        code: 200
        duration: 259.761666ms
