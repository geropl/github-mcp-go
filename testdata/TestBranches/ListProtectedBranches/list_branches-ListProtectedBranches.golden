{
  "output": "# Branches\n\nNo branches found.\n",
  "err": ""
}